tests/Tests/200__monitor_and_manage/200__metrics/203__alerts.robot

```diff
@@ -10,18 +10,38 @@ Suite Setup         Alerts Suite Setup
 
 
 *** Variables ***
-${NOTEBOOK_REPO_URL}=                   https://github.com/redhat-rhods-qe/ods-ci-notebooks-main
+${NOTEBOOK_REPO_URL}                    https://github.com/redhat-rhods-qe/ods-ci-notebooks-main
 
-${TEST_ALERT_PVC90_NOTEBOOK_PATH}=      SEPARATOR=
+${TEST_ALERT_PVC90_NOTEBOOK_PATH}       SEPARATOR=
 ...                                     /ods-ci-notebooks-main/notebooks/200__monitor_and_manage/
 ...                                     203__alerts/notebook-pvc-usage/fill-notebook-pvc-over-90.ipynb
 
-${TEST_ALERT_PVC100_NOTEBOOK_PATH}=     SEPARATOR=
+${TEST_ALERT_PVC100_NOTEBOOK_PATH}      SEPARATOR=
 ...                                     /ods-ci-notebooks-main/notebooks/200__monitor_and_manage/
 ...                                     203__alerts/notebook-pvc-usage/fill-notebook-pvc-to-100.ipynb
 
 
 *** Test Cases ***
+Verify RHODS Prometheus Alerts Are Not Firing Except For DeadManSnitch
```

Comment:
just a suggestion for the name  `Verify RHODS Prometheus Alerts Fire For DeadManSnitch Only`